### PR TITLE
Stop aliasing :file to F in the File module

### PR DESCRIPTION
**Pull request by whatyouhide**
This improves readability while still keeping the code short and concise. Single-letter aliases are hard to follow (you have to lookup the `alias` call) and IMO it is misleading to alias what looks like an Elixir module (`F`) to what is an Erlang module.

Thoughts are welcome :)

### CODE REVIEW

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by eksperimental** [comment]
:heart:
